### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/include/libtest.shlib [hunk 1]
```diff
@@ -2894,7 +2894,7 @@ function labelvtoc
 
 #
 # check if the system was installed as zfsroot or not
-# return: 0 ture, otherwise false
+# return: 0 true, otherwise false
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/include/libtest.shlib]: Done!

**Comment by freqlabs** [comment]
Please squash the commits and remember to sign-off in the commit message.

**Comment by Gelma** [comment]
> Please squash the commits and remember to sign-off in the commit message.

Ok, I try to squash in this PR and see what happen,

**Comment by Gelma** [comment]
Ok, squashed and seems correct (comparing before and after).
Please have a look

**Comment by freqlabs** [comment]
> Ok, squashed and seems correct (comparing before and after).
> Please have a look

Looks good, thanks.

File: tests/zfs-tests/include/libtest.shlib [hunk 2]
```diff
@@ -2894,7 +2894,7 @@ function labelvtoc
 
 #
 # check if the system was installed as zfsroot or not
-# return: 0 ture, otherwise false
+# return: 0 if true, otherwise false
```
Comment (rlaager) [inline tests/zfs-tests/include/libtest.shlib]: This still has a type mismatch (0 vs false). How about "return: 0 if true, non-zero if false" or "return 0 if zfsroot, non-zero if not".

File: tests/zfs-tests/tests/functional/cachefile/cachefile_004_pos.ksh [hunk 3]
```diff
@@ -38,7 +38,7 @@
 #	Verify set, export and destroy when cachefile is set on pool.
 #
 # STRATEGY:
-#	1. Create two pools with one same cahcefile1.
+#	1. Create two pools with one same cachefile1.
 #	2. Set cachefile of the two pools to another same cachefile2.
 #	3. Verify cachefile1 not exist.
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/cachefile/cachefile_004_pos.ksh]: Can we change this as well: "3. Verify cachefile1 does not exist." (adding "does")

Comment (Gelma) [reply tests/zfs-tests/tests/functional/cachefile/cachefile_004_pos.ksh]: Done!

File: tests/zfs-tests/include/libtest.shlib (hunk 2 above)
Comment (Gelma) [reply tests/zfs-tests/include/libtest.shlib]: You guys tell me what you prefer.

Comment (freqlabs) [reply tests/zfs-tests/include/libtest.shlib]:
```suggestion
# return: 0 if zfsroot, non-zero if not
```

Comment (rlaager) [reply tests/zfs-tests/include/libtest.shlib]: I like the suggestion from @freqlabs

Comment (Gelma) [reply tests/zfs-tests/include/libtest.shlib]: > I like the suggestion from @freqlabs

Done!

**Pull request merged**